src/main/java/org/sonatype/nexus/plugins/conda/internal/util/CondaPathUtils.java

```diff
@@ -55,11 +63,15 @@ private String match(final TokenMatcher.State state, final String name) {
     return result;
   }
 
+  public String buildAssetPath(final State matcherState, final String assetName) {
```

Comment:
Maybe someday we'll have a nice IT framework to add more tests. ;) Meanwhile, it might help to add some unit tests of buildAssetPath() and buildArchAssetPath() (and maybe match()) that enforce/document the intended results for various inputs.